src/modules/partition/partition.schema.yaml

```diff
@@ -6,27 +6,33 @@ $id: https://calamares.io/schemas/partition
 additionalProperties: false
 type: object
 properties:
-    efiSystemPartition: { type: string }  # Mount point
-    efiSystemPartitionSize: { type: string }
-    efiSystemPartitionName: { type: string }
+  efiSystemPartition: { type: string } # Mount point
+  efiSystemPartitionSize: { type: string }
+  efiSystemPartitionName: { type: string }
 
-    userSwapChoices: { type: array, items: { type: string, enum: [ none, reuse, small, suspend, file ] } }
-    # ensureSuspendToDisk: { type: boolean, default: true }  # Legacy
-    # neverCreateSwap: { type: boolean, default: false }  # Legacy
+  userSwapChoices:
+    {
+      type: array,
+      items: { type: string, enum: [none, reuse, small, suspend, file] },
+    }
+  luksGeneration: { type: string, enum: [luks1, luks2] }
```

Comment:
.. for the purpose of this PR, I'd prefer it if you reverted the indentation changes and just added this one line; then we can PR the indentation changes and tooling separately, if desired.